src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -883,7 +883,7 @@ protected virtual void ReportUnassigned(Symbol symbol, SyntaxNode node, int slot
                 // We've already reported the use of a local before its declaration.  No need to emit
                 // another diagnostic for the same issue.
             }
-            else if (!_alreadyReported[slot] && VariableType(symbol)?.IsErrorType() != true)
+            else if (!_alreadyReported[slot] && VariableType(symbol).IsErrorType() != true)
```

Comment:
Should check for non-null and remove the `!= true`

Reply:
Changed the `!= true`, but did not add the `null` check since I think you can only get here with a symbol that has a type.

---
In reply to: [207056599](https://github.com/dotnet/roslyn/pull/28943#discussion_r207056599) [](ancestors = 207056599)